curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md

```diff
@@ -14,6 +14,12 @@ Build a full stack JavaScript app that is functionally similar to this: <a href=
 -   Use <a href="https://replit.com/github/freeCodeCamp/boilerplate-project-exercisetracker" target="_blank" rel="noopener noreferrer nofollow">our Replit starter project</a> to complete your project.
 -   Use a site builder of your choice to complete the project. Be sure to incorporate all the files from our GitHub repo.
 
+If you were to use Replit, follow these steps to setup the project:
```

Comment:
Thanks for customizing the text for these projects. It's great that you refer to Replit as an option, then go into detail if learners go that route.

But "setup" should be in the two word verb form here. We could also simplify things a bit:

```suggestion
If you use Replit, follow these steps to set up the project:
```